app/javascript/react/components/App.js

```diff
@@ -1,7 +1,17 @@
 import React from 'react'
+import { BrowserRouter, Route, Switch } from 'react-router-dom'
+import BandsIndexPage from './bands/BandIndex'
+import BandsShowContainer from './bands/BandShowContainer'
 
 export const App = (props) => {
-  return (<h1>Make It So React</h1>)
+  return (
+    <BrowserRouter>
+      <Switch>
+        <Route exact path="/" component={BandsIndexPage} />
+        <Route exact path="/bands/:id" component={BandsShowContainer} />
```

Comment:
same note as above. not sure why there is band show page logic here?

Reply:
if you dont have a full BandsShowContainer somewhere, we should be deleting all mention of the show page until we get to that PR